ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/Transaction.java

```diff
@@ -45,6 +45,9 @@
 
   public static final BigInteger REPLAY_PROTECTED_V_BASE = BigInteger.valueOf(35);
 
+  public static final BigInteger GO_QUORUM_PRIVATE_TRANSACTION_V_VALUE_MAX = BigInteger.valueOf(37);
+  public static final BigInteger GO_QUORUM_PRIVATE_TRANSACTION_V_VALUE_MIN = BigInteger.valueOf(38);
```

Comment:
Is MAX supposed to be smaller than MIN here?

Reply:
YES

Reply:
fixed

Reply:
max and min don't make a whole lot of sense (they are never compared) but I couldn't think of anything better